### DISCUSSION TITLE
How to get similar behaviour to a file manager: e.g. dragging a file into a folder.

### DISCUSSION THREAD

**Discussion by ajmchambers** [formkit/drag-and-drop#64] [General]
Hey, my goal is to create something that behaves similar to a file manager. There are two columns: Groups and Items. Items belong to Groups and you can rearrange items within a group but also can drag items into other groups. Groups can also be reordered themselves.

| Groups | Group 1 Items     |
|:--------------|:-----------|
| **Group 1 (selected)** | Item 1       |
| Group 2 | Item 2       |
| Group 3 | Item 3       |

Goals:
- to be able to drag groups and rearrange their order.
- to be able to drag items and rearrange their order.
- to be able to drag items (from the current active group) into other groups by dragging the item on-top of the group

Here is a video example:

https://github.com/formkit/drag-and-drop/assets/5426717/aeda456e-c885-4c9f-8167-11b51425005e

Is this possible, any suggestions on how it could be done? (in particular dragging items into other groups, without those items becoming part of that list)

**Comment by sashamilenkovic** [comment]
@ajmchambers This is something I have worked on at a feature branch but not yet implemented. I'm not sure if there's going to be any sort of workaround at the moment. I can let ping back at this discussion when I get around to releasing this feature though.

**Reply by ajmchambers** [reply]
Ah ok thanks

**Comment by Lazerproof** [comment]
@sashamilenkovic Any update on this feature?

**Comment by sashamilenkovic** [comment]
@Lazerproof @ajmchambers Currently working on nested drag and drop as a feature to an upcoming project FormKit is releasing. I'll keep you posted here as to how it's coming along.

**Reply by dermitzos** [reply]
Hello. Any updates on this?

**Reply by sashamilenkovic** [reply]
@dermitzos Should be out in the next week or so.

**Reply by sashamilenkovic** [reply]
@dermitzos Hey there! Unfortunately it doesn't look like I'll get the release out today. Just wanted to give an update. Hopefully sooner rather than later.

**Reply by dermitzos** [reply]
@sashamilenkovic No worries! Thank you for the update and if I can help somehow, let me know.

**Reply by sashamilenkovic** [reply]
Appreciate that @dermitzos. New release is almost ready. I'm going to work over the weekend on the changelog and docs. Should be available on Monday 🤞

A very substantive release if I may say so myself, I think you all will be pleased.

**Reply by sashamilenkovic** [reply]
@ajmchambers @dermitzos @Lazerproof New release is out now. Still working on getting out some examples of nested drag and drop on the docs.
